lib/commands/doctor/checks/logged-in-user-owner.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const os = require('os');
+const execa = require('execa');
+const path = require('path');
+const errors = require('../../../errors');
+const chalk = require('chalk');
+const fs = require('fs');
+
+const taskTitle = 'Checking logged in user and directory owner';
+
+function checkGhostUser() {
```

Comment:
is this worth throwing in a util? Seems that it's similar to https://github.com/TryGhost/Ghost-CLI/blob/master/lib/utils/use-ghost-user.js and there could be some common code 😄

Reply:
I could refactor the existing util to have two different fn maybe? One to get the ghost uid and gid and one to check if the ghost user should be used because he owns the content directory?

Reply:
Yeah that’s what I was thinking 🙂

Reply:
This is done and updated!